Submissions/JLodha/Task_1/csvtojson.py

```diff
@@ -0,0 +1,14 @@
+import csv
+import json
+
+csvfile = open('movies.csv', encoding = "utf8",errors = 'ignore')
+fields = ['movieId','title','genres']
+reader = csv.DictReader(csvfile, fieldnames = fields)
+datalist = []
+movie_genre= 'Comedy'
+for row in reader:
+    newrow = {'movieId' : row['movieId'], 'title':row['title'], 'genres': row['genres']}
+    if(newrow['genres'] == movie_genre):
```

Comment:
There can be movies whose genre contains other genres also apart from only 'Comedy' such as 'Drama' or 'Action' so make your code more flexible.

Reply:
Updated
Please have a look at it
Thanks